packages/eden/src/client.ts

```diff
@@ -12,12 +12,12 @@ import type {
 } from './links/internal/operation'
 import { share } from './links/internal/operators'
 import type { EdenResponse } from './request'
-import type { EdenRequestParams } from './resolve'
+import type { EdenRequestParams as EdenRequestParameters } from './resolve'
 
 export type EdenSubscriptionObserver<TValue, TError> = {
   onStarted: () => void
   onData: (value: TValue) => void
-  onError: (err: TError) => void
+  onError: (error: TError) => void
```

Comment:
I actually specifically use `err` over `error` because the latter is present in some frameworks as utility functions. For example, SvelteKit and Elysia both have `error` functions that return an error response. I'm not a fan of renaming these framework utilities, so I'd prefer to just use a non-conflicting parameter name.

```ts
import { error as svelteKitError } from '@sveltejs/kit'

const options = {
  onError(error) {
    svelteKitError(error)
  }
}
```

```suggestion
  onError: (err: TError) => void
```